client/components/viewRecording.jsx

```diff
@@ -4,24 +4,47 @@ export default function RecordingView(props) {
   let recordedDate = props.recording.recordedAt.slice(0, 10);
   recordedDate = recordedDate.split('-').join('/');
   const [chosen, setChosen] = useState(false);
+  const [isDeleted, setIsDeleted] = useState(false);
   const audioClassName = chosen ? 'openAudio' : 'closeAudio';
 
   return (
-    <div className="view-container margin-0-auto font-pair wrap-container">
+    <>
+    {isDeleted
+      ? null
+      : <>
```

Comment:
This would be better with `&&` instead of `? :`.

https://reactjs.org/docs/conditional-rendering.html#inline-if-with-logical--operator